frontend/src/components/pages/components/content-editor/content-editor.tsx

```diff
@@ -52,16 +52,30 @@ export const ContentEditor: React.FC = () => {
   };
 
   const handleSaveConfirm = (): void => {
-    if (titleInputValue || markDownContent) {
-      dispatch(
-        pagesActions.editPageContent({
-          pageId: paramsId,
-          title: titleInputValue,
-          content: markDownContent,
-        }),
+    if (
+      titleInputValue &&
+      titleInputValue?.trim().length <= PageTitle.MAX_PAGE_TITLE_LENGTH
+    ) {
+      if (markDownContent || titleInputValue) {
+        dispatch(
+          pagesActions.editPageContent({
+            pageId: paramsId,
+            title: titleInputValue.trim(),
+            content: markDownContent?.length === 0 ? ' ' : markDownContent,
+          }),
+        );
+        history.push(replaceIdParam(AppRoute.PAGE, paramsId || ''));
+      }
+    } else if (titleInputValue?.trim().length === 0) {
+      toast.warning('Sorry, title could not be empty');
```

Comment:
```suggestion
      toast.error('Error: Page title cannot be empty');
```
"Sorry" sounds too unfitting for an error message

Reply:
@aleksandr-kuharenko could you please suggest what fits better here? 
Thanks

Reply:
a warning has yellow pop-up color, it looks better than a red Error pop-up as I could say

Reply:
toast.warning('Sorry, title could not be empty');
```suggestion
      toast.warning('Title could not be empty');
```